packages/fxa-auth-server/test/mail_helper.js

```diff
@@ -85,6 +85,7 @@ module.exports = (printLogs) => {
           const vsc = mail.headers['x-verify-short-code'];
           const sc = mail.headers['x-signin-verify-code'];
           const rpc = mail.headers['x-password-forgot-otp'];
+          const acv = mail.headers['x-account-change-verify-code'];
```

Comment:
Thanks for this change. Just a heads up it might conflict with Valerie's change, which is about to land.